docs/oxAuth_swagger.yaml

```diff
@@ -0,0 +1,2030 @@
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/U2fConfiguration'
+        403:
+          $ref: '#/components/responses/AccessDenied' 
+        406:
+          $ref: '#/components/responses/NotAcceptable'        
+        500:
+          $ref: '#/components/responses/InternalServerError'  
+          
+components: 
+  responses:
+    Found: #302 - FOUND
+      description: Resource Found. 
+      content: {}
+    InvalidRequest: #400 - BAD_REQUEST
+      description: Invalid parameters are provided to endpoint. 
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'  
+    Unauthorized: #401 - UNAUTHORIZED
+      description: Unauthorized access request.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    AccessDenied: #403 - FORBIDDEN
+          description: Invalid details provided hence access denied.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+    NotFound: #404 - NOT_FOUND
+          description: Resource Not Found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+    NotAcceptable: #406 - NOT_ACCEPTABLE
+          description: Request Not Acceptable.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+    InternalServerError: #500 - INTERNAL_SERVER_ERROR
+      description: Internal error occured. Please check log file for details.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+     
+  schemas:  
+    ErrorResponse:
+      required:
+      - error
+      - error_description
+      type: object
+      properties:
+        error:
+          type: string
+        error_description:
+          type: string
+        details:
+          type: string
+    GluuConfiguration:
+      required:
+      - id_generation_endpoint
+      - introspection_endpoint
+      - auth_level_mapping
+      type: object
+      properties:
+        id_generation_endpoint:
+          type: string
+        introspection_endpoint:
+          type: string
+        details:
+          type: string
+        auth_level_mapping:
+          type: object
+          additionalProperties:
+            type: object
+        scope_to_claims_mapping:
+          type: object
+          additionalProperties:
+            type: object
+    IntrospectionResponse:
+      required:
+      - active
+      - scope
+      - client_id
+      - username
+      - token_type
+      - exp
+      - iat
+      - sub
+      - aud
+      - iss
+      - jti
+      - acr_values
+      typ
```

Comment:
Please remove it, it has nothing to do with actual REST.

Reply:
I have now included the relevant IntrospectionResponse attributes in-line to  /introspection operations
![introspection_endpoint](https://user-images.githubusercontent.com/43700552/75891161-63557f00-5e55-11ea-9fd0-87ebc2567a17.JPG)